OpenSim/Common/Component.cpp

```diff
@@ -202,19 +208,25 @@ void Component::connect(Component &root)
 
 
 // Call connect on all components and find unconnected Connectors a
```

Comment:
A Component  name "validater" is important, but will be for a subsequent PR. Adding it to the TODO list (#596).